test/reputation-mining/root-hash-submissions.js

```diff
@@ -95,7 +99,8 @@ contract("Reputation mining - root hash submissions", accounts => {
   });
 
   afterEach(async () => {
-    await finishReputationMiningCycleAndWithdrawAllMinerStakes(colonyNetwork, this);
+    const shouldSetupNewInstance = await finishReputationMiningCycleAndWithdrawAllMinerStakes(colonyNetwork, this);
+    if (!shouldSetupNewInstance) await newNetworkInstance(MINER1, MINER2, MINER3);
```

Comment:
`shouldSetupNewInstance ` isn't representative of what the result is here, and so the next line doesn't read right too in that "if not shouldSetupNewInstance then setupNewInstance"..
Maybe simply rename to `success` or `reputationMiningClean`?

Reply:
You are right, `shouldSetupNewInstance` should be renamed.

Reply:
And another thing `shouldSetupNewInstance` is always coming back as `undefined` irrespective of whether `finishReputationMiningCycleAndWithdrawAllMinerStakes` succeeded or failed. This should be changed to return boolean as the check now requires.

Reply:
All Fixed!